src/__tests__/get-by-errors.js

```diff
@@ -39,6 +45,38 @@ cases(
   },
 )
 
+const GET_BY_QUERIES = [
+  'getByLabelText',
+  'getByPlaceholderText',
+  'getByText',
+  'getByAltText',
+  'getByTitle',
+  'getByRole',
+  'getByTestId',
+]
+const GET_ALL_BY_QUERIES = [
+  'getAllByLabelText',
+  'getAllByPlaceholderText',
+  'getAllByText',
+  'getAllByAltText',
+  'getAllByTitle',
+  'getAllByRole',
+  'getAllByTestId',
+]
+test.each([
+  ...[...GET_BY_QUERIES, ...GET_ALL_BY_QUERIES].reduce((acc, query) => {
+    acc.push([query, true], [query, false])
+    return acc
+  }, []),
+])(
+  '%s query will throw expected error message when config.showLogOnFail is %s',
+  (query, showLogOnFail) => {
+    configure({showLogOnFail})
+    const utils = render('<div />')
```

Comment:
total nit: try the new screen object! :)
```suggestion
    render('<div />')
```